src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js

```diff
@@ -61,11 +61,19 @@ const AddProposalPanel = React.memo(({ setProposalData }) => {
 
   const fundingMode = formData.proposalType === FUNDING_PROPOSAL
 
-  const forumRegex = new RegExp(connectedGarden.forumURL)
+  // Escaping forumURL to avoid misuse with regexp
+  const forumURLRegex = connectedGarden.forumURL.replace(
+    /[-/\\^$*+?.()|[\]{}]/g,
+    '\\$&'
```

Comment:
Also can we create a const for this regex? Maybe it will be necessary to use it elsewhere.

Reply:
Nice idea. Should we build some simply function and put where? I'll check and do that.

Reply:
I extracted to allow make the test file easily. have look @tmoutinho

Reply:
Nice 👍🏼